README.md

```diff
@@ -84,6 +84,10 @@ The following CMake options are available:
   * `OFF` (default)
   * `ON` Enable IPv6 tests. Requires that IPv6 is
     [setup](https://docs.docker.com/config/daemon/ipv6/) in Docker.
+* `USE_SANITIZER`
+   Compile with a sanitizer. Options depends on
+   [compiler](https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html#index-fsanitize_003daddress),
+   but usually: `address`, `thread`, `undefined`, `leak`
```

Comment:
I still think it's unclear that the value is one of address, thread, etc. and that the default is none.
The word "option" is ambiguous since it may refer to "USE_SANITIZER" itself. "USE_SANTITIZER" is listed under *Build options*. Thus, better use the word "value" for address, thread, etc.

Reply:
Good, I have rephrased and added some more detail now